app/controllers/sessions_controller.rb

```diff
@@ -37,34 +37,29 @@ def authorize_access!
     return true unless Octobox.restricted_access_enabled?
 
     client = Octokit::Client.new(access_token: auth_hash.credentials.token)
```

Comment:
So when you make a code block instead of using a language like `ruby` all you do is use `diff` instead 😄